src/api_server/src/config.rs

```diff
@@ -14,11 +14,26 @@ pub struct Config {
     /// Resource repository type
     pub repository_type: RepositoryType,
 
+    /// OPIONAL
     /// Resource repository description
     /// This is a JSON string,
     /// Various to repository type.
     #[serde(default)]
     pub repository_description: Option<String>,
+
+    /// OPTIONAL
+    /// Remote Attestation Service address.
+    /// Only used in remote AS mode.
+    /// If Null, default remote AS addr will be used.
+    #[serde(default)]
+    pub as_addr: Option<String>,
+
+    /// OPTIONAL
+    /// Native Attestation Service config file path
+    /// Only used in native AS mode.
+    /// If Null, default AS config will be used.
+    #[serde(default)]
+    pub as_config_file_path: Option<String>,
```

Comment:
We do not need the #[serde(default)] here as it will be deserialized to None if no this field given